crates/eldenring/src/cs/chr_ins.rs

```diff
@@ -1680,9 +1680,20 @@ pub struct PlayerIns {
     /// Set on player spawn and maybe on arena respawn?
     /// Players cannot be hurt if this is above 0.
     pub invincibility_timer_for_net_player: f32,
-    unk67c: [u8; 0x24],
+    /// Durability statuses for player's equipment (weapons and protectors only).
+    /// (DS3 leftover)
+    ///
+    /// See [crate::cs::ChrAsmSlot] for index mapping.
+    pub durability_statuses: [EquipmentDurabilityStatus; 16],
+    unk68c: u8,
+    /// Hand used for attack calculations, set by HKS.
+    pub attack_reference_hand: HandIndex,
+    /// Hand used for guard calculations, set by HKS.
+    pub guard_reference_hand: HandIndex,
+    unk698: u32,
+    unk69c: u32,
     pub player_menu_ctrl: NonNull<CSPlayerMenuCtrl>,
-    unk6b0: [u8; 0x8],
+    unk6a8: [u8; 0x8],
```

Comment:
Why did this change? Was it incorrect before?

Reply:
yeah

<img width="625" height="179" alt="image" src="https://github.com/user-attachments/assets/bb561491-b04b-470b-bdcc-1727f2323626" />